etc/kayobe/globals.yml

```diff
@@ -11,7 +11,7 @@
 # Remote path configuration (seed, seed-hypervisor and overcloud hosts).
 
 # Base path for kayobe state on remote hosts.
-#base_path:
+base_path: "{{ ansible_env.HOME/.kayobe }}"
```

Comment:
shouldnt this be:

"{{ ansible_env.HOME }}/.kayobe"

Reply:
oh yeah, good spot...  maybe it could work after all! :)